src/app/core/admin/hashtags/statistics.component.ts

```diff
@@ -0,0 +1,54 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { AsyncPipe, NgFor, NgIf, NgClass } from '@angular/common';
+import { Hashtag } from './hashtag.interface';
+import { HashtagApiService } from './hashtag-api.service';
+import { Observable } from 'rxjs';
+import { SpinnerDotsComponent } from '@shared/ui';
+import { MatTableModule } from '@angular/material/table';
+
+@Component({
+  selector: 'app-statistics',
+  standalone: true,
+  imports: [NgFor, NgIf, NgClass, AsyncPipe, SpinnerDotsComponent, MatTableModule],
+  template: `
+    <h1>Statystyki</h1>
+    <hr />
+    <div class="statistics">
+      <ng-container *ngIf="statistics | async as element; else loading">
```

Comment:
wydaje mi się, że lepiej zmienić nazwę klasy bo trochę jest mylące jeśli masz klasę `statistics` a potem zmienną `statistics`.  Klasa może być np. `statistics-container` a zmienna jako że to Observable `statistics$`

Reply:
Done